src/external/rl_gputex.h

```diff
@@ -771,7 +771,11 @@ void *rl_load_astc_from_memory(const unsigned char *file_data, unsigned int file
 // Module Internal Functions Definition
 //----------------------------------------------------------------------------------
 // Get pixel data size in bytes for certain pixel format
+#ifdef __GNUC__
+static inline int get_pixel_data_size(int width, int height, int format)
```

Comment:
Why this function needs to be `inlined`?

Reply:
Check my below answer. To remove the GCC / MinGWx64 WARNING, I posted a link below.